models/business-unit/src/division/types.ts

```diff
@@ -2,41 +2,22 @@ import type {
   BusinessUnitKeyReference,
   KeyReference,
   Store,
-  Company,
   Division,
-  CompanyDraft,
   DivisionDraft,
 } from '@commercetools/platform-sdk';
 import type { TBuilder } from '@commercetools-test-data/core';
+import type { TCompanyGraphql } from '../company/types';
 
 //BusinessUnitDraft
-export type TCompanyDraft = CompanyDraft;
 export type TDivisionDraft = DivisionDraft;
-export type TCompanyDraftBuilder = TBuilder<TCompanyDraft>;
-export type TDivisionDraftBuilder = TBuilder<TDivisionDraft>;
-export type TCreateCompanyDraftBuilder = () => TCompanyDraftBuilder;
-export type TCreateDivisionDraftBuilder = () => TDivisionDraftBuilder;
+
 //BusinessUnitDraftGraphql is only scaffolding at this time
-export type TCompanyDraftGraphql = TCompanyDraft;
-export type TDivisionDraftGraphql = TDivisionDraft;
+export type TDivisionDraftGraphql = TDivisionDraft & {
+  __typename: 'BusinessUnitDraft';
+};
```

Comment:
No `__typename` in GraphQL drafts

Reply:
Fixed with [0c72cd8](https://github.com/commercetools/test-data/pull/481/commits/0c72cd8436aafea8d4e2897fc78208e5bf5ccf05)